### PR TITLE
APEXMALHAR-2428 CompositeAccumulation for windowed operator

### PR DESCRIPTION
**Pull request by brightchen**

### CODE REVIEW

**Comment by brightchen** [comment]
@chinmaykolhatkar Please review and merge

File: library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/CompositeAccumulation.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ */
+package org.apache.apex.malhar.lib.window.accumulation;
+
+import java.util.List;
+
+import org.apache.apex.malhar.lib.window.Accumulation;
+
+import com.google.common.collect.Lists;
+
+public class CompositeAccumulation<InputT> implements Accumulation<InputT, List, List>
```
Comment (chinmaykolhatkar) [inline library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/CompositeAccumulation.java]: Can you please add @Evolving annotation to this and also the javadoc comment on when and how to use it?

**Review by chinmaykolhatkar — CHANGES_REQUESTED** [review]
All looks fine. Except one small comment. Once done I can merge this.

**Comment by brightchen** [comment]
@chinmaykolhatkar applied comments, please review

**Pull request merged**